order/app.py

```diff
@@ -115,7 +122,7 @@ def find_order(order_id):
 
     # Only return item id's
     response = order.__dict__
-    response['items'] = list(order.items.keys())
```

Comment:
Why is this change made? The docs specified as returntype for the items:  list of item ids that are included in the order, however this will return a dict

Reply:
I saw this SO thread: https://stackoverflow.com/questions/16819222/how-do-i-return-dictionary-keys-as-a-list-in-python, which suggests you could leave it out, so I tried it to satisfy my curiosity. It doesn't make a difference if you use .keys() or not